Revit_Engine/Revit_Engine.csproj

```diff
@@ -85,6 +85,9 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Library_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Physical_Engine">
+      <HintPath>..\..\..\ProgramData\BHoM\Assemblies\Physical_Engine.dll</HintPath>
```

Comment:
I'll get BHoMBot to run a report for you to identify issues with the CSProject file :smile: